### DISCUSSION TITLE
Add support of hcxdumptool

### DISCUSSION THREAD

**Discussion by pengyu** [Homebrew/discussions#380] [Casks]
https://github.com/ZerBea/hcxdumptool

It seems that this package is not found on homebrew. Can somebody added? Thanks.

**Comment by gromgit** [comment]
> Can somebody added?

Folks who ask for Homebrew additions are usually asked to add it themselves: https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request

This is especially true for a tool like `hcxdumptool`, which looks like it's both a special-purpose tool and _extremely_ sensitive to the underlying hardware and OS. Whoever adds this needs to be technically competent enough to deal with whatever issues stand in the way of producing a usable binary (I expect there will be many of those).
